test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,19 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(
+      joinNames([
+        {
+          name: "Harry",
+        },
+        {
+          name: "Ron",
+        },
+        {
+          name: "Hermione",
+        },
+      ])
+    ).toBe("Harry, Ron & Hermione");
+  });
```

Comment:
I would add more tests for edge changes e.g. if there is only 1 or 2 or more than 3 nameObjs. Currently, you return the wrong result if there is only 1 nameObj in your array.

Although you code passes for 2 and more than 3 nameObjs you should still write tests for those edge cases just to be certain of it and to catch an error if you refactor the function later.